phobos/blender/operators/editing.py

```diff
@@ -1758,7 +1758,7 @@ def poll(cls, context):
 
 class AddMotorOperator(Operator):
     """Add a motor to the selected joint.
-    It is possible to add motors to multiple joints at the same time"""
+It is possible to add motors to multiple joints at the same time"""
```

Comment:
This indentation change is faulty.

Reply:
I added this because the indentation is visible in the tooltip for the operator. With the indentation the tooltip looks like this:
![grafik](https://github.com/dfki-ric/phobos/assets/107504501/8f143305-0b31-477b-a221-aabd0af00d40)

After the change it looks like this:
![grafik](https://github.com/dfki-ric/phobos/assets/107504501/366ef791-c5c4-47cb-bb23-68b508046055)

Reply:
What happens if you add a line break before the closing """ with the correct level of indentation? Maybe this helps blender to find the correct line break. How is it done with the other operators?
Like this: 
```python
class AddMotorOperator(Operator):
    """Add a motor to the selected joint.
    It is possible to add motors to multiple joints at the same time
    """
```

Reply:
Most of them are all one liners except AddSensorOperator. How does the tooltip look there?